### PR TITLE
Do a backup when upgrading the eeprom

### PR DESCRIPTION
**Pull request by schwabe**
I am not sure if this has any other unwated side effects. 

This should help diagnose problems with the conversion.

### CODE REVIEW

File: radio/src/storage/eeprom_conversions.cpp [hunk 1]
```diff
@@ -1262,6 +1262,11 @@ bool eeConvert()
 
   ALERT(STR_STORAGE_WARNING, msg, AU_BAD_RADIODATA);
 
+// Only do a backup if supported and if it is getting a unique filename
+#if defined(EEPROM_RLC) && defined(RTCLOCK)
```
Comment (bsongis) [inline radio/src/storage/eeprom_conversions.cpp]: I would suggest SDCARD instead of RTCLOCK

Comment (schwabe) [inline radio/src/storage/eeprom_conversions.cpp]: Yes, but currently eeprombackup is only defined if EEPROM_RLC is defined. Also the 480x272 radio_version.cpp lacks the backup functionality. I think backup might not have been implemented for Horus yet.

**Comment by kilrah** [comment]
Is the SD card even mounted at this point?

**Comment by schwabe** [comment]
I think I tested it but I can test again to be sure.

**Comment by schwabe** [comment]
okay checked the code. Seems sd card init is after this.

**Comment by bsongis** [comment]
you could perhaps mount the SD card just before calling this function

**Comment by schwabe** [comment]
yeah but let me test that on real hardware in order to be sure. I will reopen the pull request when I have a working version

**Pull request closed without merging**